src/panels/lovelace/entity-rows/hui-button-entity-row.ts

```diff
@@ -0,0 +1,82 @@
+import "@material/mwc-button/mwc-button";
+import {
+  css,
+  CSSResultGroup,
+  html,
+  LitElement,
+  PropertyValues,
+  TemplateResult,
+} from "lit";
+import { customElement, property, state } from "lit/decorators";
+import { UNAVAILABLE } from "../../../data/entity";
+import { HomeAssistant } from "../../../types";
+import { hasConfigOrEntityChanged } from "../common/has-changed";
+import "../components/hui-generic-entity-row";
+import { createEntityNotFoundWarning } from "../components/hui-warning";
+import { ActionRowConfig, LovelaceRow } from "./types";
+
+@customElement("hui-button-entity-row")
+class HuiButtonEntityRow extends LitElement implements LovelaceRow {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @state() private _config?: ActionRowConfig;
+
+  public setConfig(config: ActionRowConfig): void {
+    if (!config) {
+      throw new Error("Invalid configuration");
+    }
+    this._config = config;
+  }
+
+  protected shouldUpdate(changedProps: PropertyValues): boolean {
+    return hasConfigOrEntityChanged(this, changedProps);
+  }
+
+  protected render(): TemplateResult {
+    if (!this._config || !this.hass) {
+      return html``;
+    }
+
+    const stateObj = this.hass.states[this._config.entity];
+
+    if (!stateObj) {
+      return html`
+        <hui-warning>
+          ${createEntityNotFoundWarning(this.hass, this._config.entity)}
+        </hui-warning>
+      `;
+    }
+
+    return html`
+      <hui-generic-entity-row .hass=${this.hass} .config=${this._config}>
+        <mwc-button
+          @click=${this._pressButton}
+          .disabled=${stateObj.state === UNAVAILABLE}
+        >
+          ${this.hass.localize("ui.card.button.press")}
```

Comment:
I think that this row can be like this:
- name: Device name
- button label: entity name

Reply:
And if the entity doesn't belong to a device?

Reply:
Then we use "press" ? 🤷 

But right, it is weird…

Reply:
I think we should use press now... we could use device classes to influence this maybe?
For example, a DEVICE_CLASS_RESTART for this one, to show "RESTART" and ask for confirmation for the future?

Reply:
ok.